src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,15 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+Integer nvim_get_caps_state(Error *err )
+  FUNC_API_SINCE(4)
+{
+  int res = capslock_status();
```

Comment:
Missing `const`. And a lot of style guide violations, though most should be reported.

Reply:
Yes, it was presently just for the check of working so no serious attempts for this thing presently was made, so if I make it correct I will do it, I wanted to see the run of the travis and appveor